File: app/src/main/java/com/t895/freebie/utils/RoundedCornerHelper.java
package com.t895.freebie.utils;

import android.content.Context;
import android.util.DisplayMetrics;

public class RoundedCornerHelper
{
  public static final int EIGHT_DP = 8;

  public static int dpToPx(Context context, int dp)
  {
    DisplayMetrics displayMetrics = context.getResources().getDisplayMetrics();
    return Math.round(dp * (displayMetrics.xdpi / DisplayMetrics.DENSITY_DEFAULT));
  }
}

File: app/src/main/java/com/t895/freebie/utils/RoundedCornerHelper.kt
package com.t895.freebie.utils

import android.content.Context
import android.util.DisplayMetrics
import kotlin.math.roundToInt

object RoundedCornerHelper
{
    const val EIGHT_DP = 8

    fun dpToPx(context: Context, dp: Int): Int
    {
        val displayMetrics = context.resources.displayMetrics
        return (dp * (displayMetrics.xdpi / DisplayMetrics.DENSITY_DEFAULT)).roundToInt()
    }
}
